File: API/src/Modules/Users/Momentum.Users.Application/DTOs/ProfileDto.cs
using System;

namespace Momentum.Users.Application.DTOs
{
    public class ProfileDto
    {
        public DateTime CreatedAt { get; set; }
        public DateTime? UpdatedAt { get; set; }
        public Guid UserId { get; set; }
        public string Bio { get; set; }
    }
}

File: server/src/dto/user/profile.dto.ts
import { Profile } from "@prisma/client";

export class ProfileDto {
	id: number;
	bio: string;	
    createdAt: Date;
    updatedAt: Date;
	userID: number;
	featuredBadgeID: number;
	
	constructor(
		_profile: Profile
	) {
		this.bio = _profile.bio;
		this.userID = _profile.userID;
		this.featuredBadgeID = _profile.featuredBadgeID;
	}
}

